Tests/LeafKitTests/TestHelpers.swift

```diff
@@ -0,0 +1,267 @@
+import XCTest
```

Comment:
Various helper extensions, functions, variables, etc that are repeatedly used for Testing are isolated to this file to avoid sprawl, as well as tests that are purely internal conveniences (eg, PrintTests)